pom.xml

```diff
@@ -215,6 +231,23 @@
 	    <phase>validate</phase>
             <goals><goal>enforce</goal></goals>
           </execution>
+          <execution>
```

Comment:
This seems like a good check... although with latest chnages, should `phase` be `verify` (I change this for jacoco plugin recently)?